### DISCUSSION TITLE
Component - how to find out there is a new version

### DISCUSSION THREAD

**Discussion by balrog-nona** [OpenIndiana/oi-userland#6291] [Q&A]
Hello everybody,

I would like to ask where or how do you get the information about component having a new version? So far, I usually go through components and randomly choose which to examine. If I am lucky, I discover that there is a new version and therefore I can update it here.
Is there a better strategy? 
Thank you :)

**Comment by AndWac** [comment]
Am 10.12.20 um 18:00 schrieb Nona Hansel:
>
> Hello everybody,
>
> I would like to ask where or how do you get the information about
> component having a new version? So far, I usually go through
> components and randomly choose which to examine. If I am lucky, I
> discover that there is a new version and therefore I can update it here.
> Is there a better strategy?
> Thank you :)
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenIndiana/oi-userland/discussions/6291>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADET4NZ4BEUJTY7PXPBULJDSUD5CVANCNFSM4UVJSY4Q>.
>
That's easy: https://repology.org/repository/openindiana

You should only be aware that there are at least two kinds of false
positives:
1. Some distributions create arbitrary version numbers
2. IPS cannot cope with alphanumerical version numbers and leading
zeroes so the IPS_VERSION_NUMBER has been introduced and Repology cannot
cope with that

Regards,
Andreas

**Reply by balrog-nona** [reply]
Thank you, Andreas, that's really heplfull!
